contributing.md

```diff
@@ -1,126 +1,179 @@
-decentralized identity-related and -adjacent communities. Ideally, these inputs
-will lead to issues for consensual/deliberative resolution. Further
-validation and improvement of the section can/expected to be done via (code) 
-implementations and testing. 
-
-During periods of **normal mode** development, issue-linked PRs should allow up 
-to 5 business day for review and change requests.  During periods of **crunch 
-mode** development, this window is reduced to 2 business days. At least two 
-maintainers should still review all PRs.
-
-
-#### Assigning section content in the STABLE state: 
-
-When all of the following conditions are met: 
-* There is quality code that demonstrates the concepts it describes.
-* There are no outstanding issues related to that section that the Working Group 
-Participants consider substantive.
-* The Working Group has at least 2 conformant implementations, at least 1 of
-which originated outside of the Working Group 
-
-When the WG has consensus that the above conditions have been reached, 
-the section enters a final 2-week comment period if no substantive
-objections are raised against the LAST-COMMENTS status. The start
-of this period, intended for "last comments," should be announced
-on the Working Group mailing list. If substantial discussion or change
-happens, this LAST-COMMENTS phase can be extended 1 additional
-week at the discretion of the chairs.
-
-Once a section reaches STABLE status, non-substantive changes can be merged at 
-maintainer discretion, however, substantive changes are generally discouraged 
-except by discretion of the chairs. Even then, substantial changes should be 
-discussed on a Working Group call. If resolved asynchronously via GitHub issues,
-the discussion during a Working Group call may be brief.
-
-Note:  when it comes to substantive changes, there are alternatives to the 
-above, which may delay completion of the consensual work: 
-1. document the change in an issue but make no change to the section
-2. write an extension standard (which may break some aspects of 
-original/trunk spec)
-3. depreciate current version of a standard, and write a new version together
-as a group or in a rechartered smaller group
-
-Once all expected sections of the Scope reach the STABLE stage the Working 
-Group commences the work carried out during the Draft stage to Working Group
-Approved stage as described in DIF’s Charter.
+given time. Maintainers *MUST* have different organizational affiliations to
+represent the diversity of the Working Group (i.e., no two maintainers may work
+for the same company, organization, or project).
+
+## Work Mode and Process
+
+### Process Summary
+The following is a summary of the process used by the Claims and Credentials
+Working Group for the Presentation Exchange Specification. Greater detail may
+be found below.
+1. The group determines the version number of the next release of the
```

Comment:
All 1?

Reply:
markdown should render them as ordered numerical bullets, so I am in the habit of always using 1s so I never have to re-number bullet points when adding to or reconfiguring numbered lists